README.md

```diff
@@ -5,8 +5,9 @@
 ### What is this repository for? ###
 Having access to the source code you can:
 
-* Add new weapons, NPCs, fix bugs and generally add new features to the game and submit them as pull requests 
-* Create mods. Both server side and client side mods. Or fully standalone mods 
```

Comment:
This shouldn't be removed. We still allow creating mods from our source. If you want to make it clear that server and client mods shouldn't be used to cheat leaderboards or cheat on ranking servers then this can be added as a separate line.